### PR TITLE
Update Windows section

### PR DESCRIPTION
**Pull request by peterkokot**
This patch adds a quick introduction of Cash - a cross-platform implementation of Unix shell commands

### CODE REVIEW

**Comment by jlevy** [comment]
Cool. @dspinellis review?

File: README.md [hunk 1]
```diff
@@ -554,6 +554,8 @@ These are items relevant *only* on OS X.
 
 - You can perform and script most Windows system administration tasks from the command line by learning and using `wmic`.
 
+- Another option to get Unix look and feel under Windows is [Cash](https://github.com/dthree/cash).
```
Comment (dspinellis) [inline README.md]: The *cash* idea is very interesting! A disclaimer needs to be added regarding the available functionality; something like: "Note that only very few Unix commands and command-line options are available in this environment".

While at it consider also adding [MinGW](http://www.mingw.org/) with an corresponding disclaimer, and a few bullets guiding people to select each of the three options.  The *Cygwin* bullet should then be merged into this section.

**Comment by peterkokot** [comment]
PR updated with:
* A note as suggested by @dspinellis about Cash
* Introduction to [MinGW](http://www.mingw.org/) and [MSYS](http://www.mingw.org/wiki/msys)
* And since Windows 10 a nicely done native way to access the Unix command line via the Bash on Ubuntu on Windows and run Linux programs.

If there are other suggestions or improvements need to be made let me know please.

**Comment by jlevy** [comment]
Great additions! Agree bash on windows is significant and should be mentioned. Will await @dspinellis review.

File: README.md [hunk 2]
```diff
@@ -540,6 +540,10 @@ These are items relevant *only* on OS X.
 
 ## Windows only
 
+These items are relevant *only* on Windows.
+
+- Since Window 10 use [Bash on Ubuntu on Windows](https://msdn.microsoft.com/commandline/wsl/about), which provides a familiar Bash environment with Unix command line utilities and allows Linux programs to run on Windows.
```
Comment (jlevy) [inline README.md]: "Since Windows 10, ..." (needs comma and s)

File: README.md [hunk 3]
```diff
@@ -556,6 +560,10 @@ These are items relevant *only* on OS X.
 
 - You can perform and script most Windows system administration tasks from the command line by learning and using `wmic`.
 
+- Another option to get Unix look and feel under Windows is [Cash](https://github.com/dthree/cash). Note that only very few Unix commands and command-line options are available in this environment.
+
+- Alternative option to get GNU developer tools (such as GCC) on Windows, is [MinGW](http://www.mingw.org/) and its [MSYS](http://www.mingw.org/wiki/msys) package that provides utilities such as bash, gawk, make and grep. MSYS doesn't have all the features compared to Cygwin.
```
Comment (jlevy) [inline README.md]: "An alternative..." (add "An")
"on Windows is" (no comma)
"package, which provides"

**Comment by jlevy** [comment]
Added a few minor language corrections, too.

File: README.md [hunk 4]
```diff
@@ -540,6 +540,10 @@ These are items relevant *only* on OS X.
 
 ## Windows only
 
+These items are relevant *only* on Windows.
+
+- Since Windows 10, you can use [Bash on Ubuntu on Windows](https://msdn.microsoft.com/commandline/wsl/about), which provides a familiar Bash environment with Unix command line utilities and allows Linux programs to run on Windows.
+
```
Comment (dspinellis) [inline README.md]: * On Windows 10, you can
* command line utilities. On the plus side, this allows Linux programs to run on Windows. On the other hand this does not allow Windows programs to run from the Bash prompt.

File: README.md [hunk 5]
```diff
@@ -556,6 +560,10 @@ These are items relevant *only* on OS X.
 
 - You can perform and script most Windows system administration tasks from the command line by learning and using `wmic`.
 
+- Another option to get Unix look and feel under Windows is [Cash](https://github.com/dthree/cash). Note that only very few Unix commands and command-line options are available in this environment.
+
+- An alternative option to get GNU developer tools (such as GCC) on Windows is [MinGW](http://www.mingw.org/) and its [MSYS](http://www.mingw.org/wiki/msys) package, which provides utilities such as bash, gawk, make and grep. MSYS doesn't have all the features compared to Cygwin.
```
Comment (dspinellis) [inline README.md]: I suggest to add: MinGW is particularly useful for creating native Windows ports of Unix tools.

**Comment by peterkokot** [comment]
@jlevy @dspinellis added as suggested :+1:

**Comment by jlevy** [comment]
Are we good to merge? @dspinellis ?

**Comment by dspinellis** [comment]
LGTM; merge!

**Comment by jlevy** [comment]
👍 Thanks to you both!

**Pull request merged**